### PR TITLE
fmt: improve single line const comment placement

### PR DESCRIPTION
**Pull request by Larpon**
This PR improves the placement of tail comments after *single line* const expressions.

Before this PR, the behaviour was:

`input.v`
```v
// leave 1.0
const one = 1 // leave 1.1
// move me 1.2
const two = 2 /* move 2
*/
```

`input.fmt.v`
```v
// leave 1.0
const one = 1

// leave 1.1
// move me 1.2

const two = 2

/*
move 2
*/
```

After this PR:

`input.v`
```v
// leave 1.0
const one = 1 // leave 1.1
// move me 1.2
const two = 2 /* move 2
*/
```

`input.fmt.v`
```v
// leave 1.0
const one = 1 // leave 1.1

// move me 1.2
const two = 2

/*
move 2
*/
```

... I find the behaviour introduced with this PR a bit more logical.
The most important part, to me, is that the trailing comment after the single line const expression is kept:
`const one = 1 // leave this comment`
`const two = 2 /* also leave this comment - but rewrite to use "//" */`

```
const three = 3 /* but
move this */
```

So basically it fixes it to have same behaviour as comments after `type X = y // <comment>` - which is nice

### CODE REVIEW

**Comment by Larpon** [comment]
Looks like `vtl` needs updating after #13570

**Comment by Larpon** [comment]
... now it's Gitly :sweat_smile:

**Comment by Larpon** [comment]
Failing tcp test in sanitized CI should be unrelated

**Comment by Larpon** [comment]
Fixes #11396

**Comment by Larpon** [comment]
Ready
